src/input/hotkey.c

```diff
@@ -381,25 +381,25 @@ static void handle_bookmark(int number, int with_modifier)
     }
 }
 
-static void take_screenshot(void)
+static void take_screenshot(int full_city)
 {
-    graphics_save_screenshot();
+    graphics_save_screenshot(full_city);
 }
 
-void hotkey_func(int f_number, int with_modifier)
+void hotkey_func(int f_number, int with_ctrl)
```

Comment:
This loses the Shift+F1 to bookmark functionality. Please put the "with_modifier" back (and perhaps rename it to "with_any_modifier")